opendrive2lanelet/io/extended_file_writer.py

```diff
@@ -1,3 +1,18 @@
+
+# Copyright (C) 2020-2021 LEIDOS.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
```

Comment:
We do not have copyright on this file. Only the files I mark as ours should include this copyright notice.